.github/workflows/depviz.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Can you add the uses here instead of in the `depviz-action`, itself?

```
- uses: actions/checkout@v3
```

Also, can you split right now the depviz action into two commands, one for fetch and one for generate, so we can make a single fetch and multiple generations?